packages/boba/gateway/src/components/listToken/listToken.js

```diff
@@ -186,7 +186,7 @@ function ListToken({ token, chain, networkLayer, disabled, loading,
                       tooltip="A multi-chain bridge to Alt L1."
                       fullWidth
                     >
-                      Bridge
+                      Bridge to alt L1
```

Comment:
I would drop the alt here
```suggestion
                      Bridge to L1
```